app/controllers/api/v1/restrooms.rb

```diff
@@ -46,7 +46,9 @@ class Restrooms < Grape::API
           r = Restroom
           r = r.accessible if params[:ada].present?
           r = r.unisex if params[:unisex]
-          paginate(r.near([params[:lat], params[:lng]], 20, :order => 'distance'))
+          restrooms = r.near([params[:lat], params[:lng]])
```

Comment:
I think my preference here would be to just have a `restrooms.near` that composes the active record and yelp `near` requests. That way `near` can be swapped out with any implementation regardless of search backend